docs/_templates/layout.html

```diff
@@ -22,5 +22,13 @@
         See <a class="reference external" href="https://zulip.readthedocs.io/en/stable/{{ pagename }}.html">documentation for the latest stable release</a>.</p>
     </div>
     {% endif %}
+
+    <script type="text/javascript">
+      // <!-- Adds target=_blank to external links -->
+      $(document).ready(function () {
+        $('a[href^="http://"], a[href^="https://"]').not('a[class*=internal]').attr('target', '_blank');
```

Comment:
I would prefer to also add attribute `rel="noopener noreferrer"` when using `target="_blank"`. You can read about it [at this stackoverflow link](https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable#:~:text=Add%20rel%3D"noopener"%20or,performance%20and%20prevent%20security%20vulnerabilities.&text=rel%3D"noopener"%20prevents%20the,able%20to%20access%20the%20window.) why.

Reply:
Cool, sounds like a good idea - the answers have convinced me. Will edit and push, thanks for the review :) Also - I have mentioned the linked issue when creating the PR, but it showing up as linked to the main issue. Any idea why?

Reply:
I've added the change. New test results ./tools/test-documentation (everything passed).
[test-documentation.txt](https://github.com/zulip/zulip/files/6650254/test-documentation.txt)